src/test/java/ioananadroflorea/tests/Tema5/DemoQA.java

```diff
@@ -0,0 +1,111 @@
+package ioananadroflorea.tests.Tema5;
+
+import ioanaandroflorea.driver.BrowserManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.Keys;
+
+public class DemoQA {
+    public static void main(String[] args) {
```

Comment:
Make sure that you have no exceptions when running this method. Thanks!

Reply:
I have no problem in running this method. I don't know what to do. You can explain me?

Reply:
Look in the console. Don't you have an Exception there?